src/hooks/simulation/useSetCurrentSimulationFromParams.ts

```diff
@@ -50,6 +50,8 @@ export function useSetCurrentSimulationFromParams() {
     // if the simulation is not in the localStorage, we add it
     initSimulation(simulation)
     setIsCorrectSimulationSet(true)
+
+    window.location.href = pathname
```

Comment:
Commentaire ?

Reply:
No idea de pourquoi j'ai fait ça. J'imagine que c'est important

Reply:
Oh, raison de plus pour ajouter au minimum un commentaire : `TODO: Weird code, should investigate when possible`

Reply:
J'ai ajouté un commentaire pour faire croire que je sais ce que je fais

Reply:
C'est tout ce qu'on te demande Flocon !